README.md

```diff
@@ -154,6 +158,77 @@ Please use our [documentation site](https://data-driven-forms.org/). In case of
 - Examples of schemas (PatternFly 3)
   - [ManageIQ Form Components](https://github.com/ManageIQ/manageiq-ui-classic/tree/master/app/javascript/components)
 
+### Development setup
+
+Data Driven Forms is a monorepo which uses [Lerna](https://github.com/lerna/lerna), so you can use all its commands as well.
+
+1. Install
+
+```console
+yarn install
+```
+
+2. Build
+
+```console
+yarn build
+```
+
+3. Run a package
+
+Each package has a small playground `package/demo`, where you can test your changes.
+
+```console
+cd packages/pf3-component-mapper
+yarn start
+```
+
+4. How to clean?
+
+```console
+rm yarn.lock
+yarn lerna clean # will delete all node_modules
+```
+
+All packages are linked together by default, so if you run a `yarn build` in a package, all other packages are updated to the latest version of that package.
+
+#### Tests
+
+Tests needed to be run from the core folder.
+
+```console
+yarn test
+
+yarn test packages/pf3-component-mapper
+```
+
+#### Commits
+
+Data Driven Forms uses [Semantic Release](https://github.com/semantic-release/commit-analyzer)
+
+Format:
+
+```
+[type]([package]): message
+
+fix(pf3): title accepts node
+```
+
+Types:
+- `feat`: a new feature, will trigger new `_.X._` release
+- `fix`: a fix, will trigger new `_._.X` release
+
+Packages:
+- Please describe which package is being changed `pf3`, `renderer`, ...
+
+Please, do not use Semantic Release, if you update only the demo.
+
+All packages are releasing together and they share the version number.
```

Comment:
I would maybe add an examaple of a commit message here

Reply:
It is a few lines above: `fix(pf3): title accepts node`